src/App.js

```diff
@@ -1,34 +1,38 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.css';
 import NameBlock from './NameBlock';
-import testData from './testData.json';
 
 function App() {
 
   const billGatesTotal = 6379000000;
-  let leftover = billGatesTotal;
 
-  const data = testData.people;
+  const [submissions, setSubmissions] = useState([]);
+  const [totalDebt, setTotalDebt] = useState(0);
 
-  function getLeftover(){
-    data.forEach((person) => {
-      leftover -= person.debt;
-    });
 
-  }
+  useEffect(() => {
+    fetch("https://tpkfcvx8jf.execute-api.us-east-1.amazonaws.com/dev/submissions")
```

Comment:
Can you move this into a configuration file? We'll have different urls for dev and prod and will want to be able to choose which one to build with :)

Reply:
I moved them to env files, will that work?

Reply:
Yepo!